tests/unit/cpp_api.cpp

```diff
@@ -419,3 +419,19 @@ TEST_F(CppApiTestFixture, TestDuration) {
   // Use Value move constructor
   auto value_y = mgp::Value(mgp::Duration("PT2M2.33S"));
 }
+
+TEST_F(CppApiTestFixture, TestNodeProperties) {
+  mgp_graph raw_graph = CreateGraph(memgraph::storage::View::NEW);
+  auto graph = mgp::Graph(&raw_graph);
+
+  auto node_1 = graph.CreateNode();
+
+  ASSERT_EQ(node_1.Properties().size(), 0);
+
+  std::map<std::string, mgp::Value> node1_prop = node_1.Properties();
+  node_1.SetProperty("b", mgp::Value(2.0));
+
+  ASSERT_EQ(node_1.Properties().size(), 1);
+  ASSERT_EQ(node_1.Properties()["b"].ValueDouble(), 2.0);
+  ASSERT_EQ(node_1.GetProperty("b").ValueDouble(), 2.0);
```

Comment:
If the type of the property doesn't matter, could you please use something that is more predictable than double? Maybe on some architectures the equality check might not fail for some precision related reasons.

Reply:
Of course :)